app/components/addRepository/add.repository.component.ts

```diff
@@ -44,6 +42,10 @@ export class AddRepositoryComponent {
     }
   }
 
+  getRepos() {
```

Comment:
Is it possible to add a comment here that give a bit more information of how it's being used?
Say for example, "Allows retrieval of recent repository list in array form for display." Or something. Just for future reference purposes since its a new function.

Reply:
Yes, I can definitely do that.